app/gui/src/project-view/components/visualizations/TableVisualization/tableVizToolbar.ts

```diff
@@ -258,6 +267,37 @@ function useSortFilterNodesButton({
     } else if (sortModelValue.length) {
       patterns.push(getAstPatternSort())
     }
+
+    function getRemoveColumnsAstPattern() {
+      const columns = columnsToRemove.map((col) => Ast.TextLiteral.new(col))
```

Comment:
It's better to provide the module when constructing any AST objects--it will work without it, but it's unfortunately pretty inefficient. There's a helper to simplify the common case of building a Vector of expressions (all in the same module) from an array of values: `Ast.Vector.build(columnsToRemove, Ast.TextLiteral.new, ast.module)`